test/junos/mocked_data/test_get_bgp_config/normal/expected_result.json

```diff
@@ -1 +1,86 @@
-{"PEERS-GROUP-NAME": {"neighbors": {"192.168.0.1": {"export_policy": "", "prefix_limit": {"inet": {"unicast": {"limit": 100}}}, "route_reflector_client": false, "description": "Facebook [CDN]", "local_as": 0, "nhs": false, "local_address": "", "remote_as": 32934, "authentication_key": "", "import_policy": ""}, "172.17.17.1": {"export_policy": "", "prefix_limit": {"inet": {"unicast": {"limit": 500}}}, "route_reflector_client": false, "description": "Twitter [CDN]", "local_as": 0, "nhs": false, "local_address": "", "remote_as": 13414, "authentication_key": "", "import_policy": ""}}, "export_policy": "PUBLIC-PEER-OUT", "multipath": true, "prefix_limit": {}, "description": "", "local_as": 13335, "multihop_ttl": 0, "apply_groups": ["B", "G", "P", "-", "P", "R", "E", "F", "I", "X", "-", "L", "I", "M", "I", "T"], "remote_as": 0, "remove_private_as": true, "local_address": "", "type": "external", "import_policy": "PUBLIC-PEER-IN"}}
+{
+  "_": {
+    "export_policy": "",
+    "multipath": false,
+    "prefix_limit": {},
+    "description": "",
+    "local_as": 13335,
+    "multihop_ttl": 0,
+    "apply_groups": [],
+    "remote_as": 0,
+    "remove_private_as": false,
+    "local_address": "",
+    "type": "",
+    "import_policy": "",
+    "neighbors": {}
+  },
+  "PEERS-GROUP-NAME": {
+    "neighbors": {
+      "192.168.0.1": {
+        "export_policy": "",
+        "prefix_limit": {
+          "inet": {
+            "unicast": {
+              "limit": 100
+            }
+          }
+        },
+        "route_reflector_client": false,
+        "description": "Facebook [CDN]",
+        "local_as": 0,
+        "nhs": false,
+        "local_address": "",
+        "remote_as": 32934,
+        "authentication_key": "",
+        "import_policy": ""
+      },
+      "172.17.17.1": {
+        "export_policy": "",
+        "prefix_limit": {
+          "inet": {
+            "unicast": {
+              "limit": 500
+            }
+          }
+        },
+        "route_reflector_client": false,
+        "description": "Twitter [CDN]",
+        "local_as": 0,
+        "nhs": false,
+        "local_address": "",
+        "remote_as": 13414,
+        "authentication_key": "",
+        "import_policy": ""
+      }
+    },
+    "export_policy": "PUBLIC-PEER-OUT",
+    "multipath": true,
+    "prefix_limit": {},
+    "description": "",
+    "local_as": 13335,
+    "multihop_ttl": 0,
+    "apply_groups": [
+      "B",
+      "G",
+      "P",
+      "-",
+      "P",
+      "R",
+      "E",
+      "F",
+      "I",
+      "X",
+      "-",
+      "L",
+      "I",
+      "M",
+      "I",
+      "T"
+    ],
```

Comment:
this absolutely looks like a bug, but I am just reformatting/refactoring the file here.

Reply:
Yeah that is a bug...looks like we have a case where we expected it to be a list or a dict and looped over it (and it ended up being a string instead).

Reply:
Do we need to create a separate issue for this?